pipelines/consolidate_injected_deepCoadd_catalogs.yaml

```diff
@@ -0,0 +1,4 @@
+description: Consolidation of per-band and per-patch catalogs into tract-level catalogs.
+tasks:
+  consolidate_injected_catalogs:
+    class: lsst.source.injection.utils.consolidate_injected_deepCoadd_catalogs.ConsolidateInjectedCatalogsTask
```

Comment:
Can this all be put into a single YAML with the new tasks? I.e., after we've finished injecting, we could run all three tasks together, right? We can add subsets for `consolidate` and `match`, if you want to keep them separate somehow.